src/test/kotlin/com/appland/appmap/AppmapPluginTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.appland.appmap
+
+import org.gradle.api.Project
+import org.junit.Assert.*
+import org.junit.Test
+import org.gradle.testfixtures.*
+
+class AppmapPluginTest {
+
```

Comment:
@dustinbyrne this test is already passing, meaning the plugin is already being applied and "working"